08-transport.Rmd

```diff
@@ -117,24 +125,24 @@ These zone-level data are small but often vital for gaining a basic understandin
 
 ## Transport zones
 
-Although transport systems are primarily based on linear features and nodes --- including pathways and stations --- it often makes sense to start with areal data.
-Three types of zone will typically be of particular interest: the study region, origin zones (typically residential areas), and destination zones (which are often the same as the origin zones).
+Although transport systems are primarily based on linear features and nodes --- including pathways and stations --- it often makes sense to start with areal data <!-- source and/or example -->.
+Three zone types will typically be of particular interest: the study region, origin zones (typically residential areas), and destination zones (which are often the same as the origin zones). <!-- this implies people are going from their origin to the origin possibly via a detour (which might be work/supermarket/etc.--> 
```

Comment:
After reading on I got what you were referring to but here it is a bit confusing to say that origin and destination zones are the sames when saying that origin zones are typically residential areas. Do you know what I mean?

Reply:
Yep - potentially confusing. Can you think of a less confusing way of phrasing this?